bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/MetricsService.java

```diff
@@ -65,6 +66,7 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
             statsProvider.writeAllMetrics(writer);
             writer.flush();
             response.setCode(StatusCode.OK);
+            response.setContentType(TextFormat.CONTENT_TYPE_004);
```

Comment:
we really have to add a new jar only for a string constant ?
I think that it is overkill

can we drop the jar and use a normal string here ?

Reply:
I can revert those changes, are the maintainers in agreement? @hangc0276

Reply:
Sorry, I forgot it will introduce a new dependency. I agree to revert those changes.